### DISCUSSION TITLE
Conversion mensaje JSON a HL7

### DISCUSSION THREAD

**Discussion by 5011-5011** [nextgenhealthcare/connect#5336] [Feature/Enhancement Ideas]
Debe llegar a mi canal un mensaje en formato JSON desde una API, y debo transformar el mensaje a formato HL7v2. Agradeceria cualquier aporte  o idea.
Gracias

**Comment by pacmano1** [comment]
What have you tried?  There are YouTube videos that demonstrate similar concepts. 

https://youtu.be/GfpHd8_u6mk

**Reply by 5011-5011** [reply]
Habia creado un transformador, pero no me funciono. Tambien me base en 1 video que habia visto. El que compartiste tu esta mucho mejor, muchisimas gracias!
